src/PlatformThreads.h

```diff
@@ -25,6 +25,17 @@ typedef struct _PLT_THREAD {
     void *context;
     bool alive;
 } PLT_THREAD;
+#elif defined(__WIIU__)
+typedef OSMutex PLT_MUTEX;
+typedef struct _PLT_EVENT {
+    OSMutex mutex;
+    OSCondition cond;
+    int signalled;
+} PLT_EVENT;
+typedef struct _PLT_THREAD {
+    OSThread thread;
+    int cancelled;
+} PLT_THREAD;
```

Comment:
Based on my (very limited) look at the Wii U headers, I think we could use a `OSFastMutex` and `OSFastCondition` for `PLT_MUTEX` and `PLT_EVENT`. We don't need recursive locking or anything fancy like that.

What are your thoughts?

Reply:
Yeah, you're right.
The fast versions should work fine for this.